File: Min. USB Devices Ever Mounted/usb.go
/*
    Minimum number of USB devices ever mounted, Go
    Module written by Brandon Arvanaghi 
    Website: arvanaghi.com 
    Twitter: @arvanaghi
*/

package main

import (
  "fmt"
  "strconv"
  "os"
  "golang.org/x/sys/windows/registry"
)

func main() {
  MinimumUSBHistory := 2

  if (len(os.Args) >= 2) {
      MinimumUSBHistory, _ = strconv.Atoi(os.Args[1])
  }

  Opened_Key, err := registry.OpenKey(registry.LOCAL_MACHINE, `SYSTEM\ControlSet001\Enum\USBSTOR`, registry.QUERY_VALUE)
  defer Opened_Key.Close()

  keyInfo, err := Opened_Key.Stat()

  if (err == nil) {
      if (int(keyInfo.SubKeyCount) >= MinimumUSBHistory) {
          fmt.Println("Proceed!")
      } else {
          fmt.Println("Number of USB devices ever mounted: ", keyInfo.SubKeyCount)
      }
  } 
}

File: Min. USB Devices Ever Mounted/usb.py
#
#   Minimum number of USB devices ever mounted, Python
#   Module written by Brandon Arvanaghi
#   Website: arvanaghi.com 
#   Twitter: @arvanaghi
#

from winreg import *
import sys

MinimumUSBHistory = 2

if len(sys.argv) == 2:
	MinimumUSBHistory = int(sys.argv[1])

HKLM = ConnectRegistry(None,HKEY_LOCAL_MACHINE)
Opened_HKLM_Key = OpenKey(HKLM, r'SYSTEM\ControlSet001\Enum\USBSTOR')

if QueryInfoKey(Opened_HKLM_Key)[0] >= MinimumUSBHistory:
	print("Proceed!")
else:
	print("Number of USB devices ever mounted: " + str(MinimumUSBHistory))